src/components/scenes/SettingsScene.tsx

```diff
@@ -24,15 +24,14 @@ import { config } from '../../theme/appConfig'
 import { connect } from '../../types/reactRedux'
 import { NavigationProp } from '../../types/routerTypes'
 import { secondsToDisplay } from '../../util/displayTime'
-import { Collapsable } from '../common/Collapsable'
```

Comment:
This component isn't used anywhere else, so you could delete `Collapsable.tsx` as well.

Reply:
I disagree, this component is well implemented as a simple roll-in, roll-out transition for short-lived components; eg the scam warning card in the Send Scene, which to me currently seems too solidified a part of the scene... before vanishing after pasting an address.